### DISCUSSION TITLE
Ideas to help TPS

### DISCUSSION THREAD

**Discussion by ReiTheGoat01** [Stardust-Labs-MC/StardustMC#32] [Ideas]
An idea I had is to remove phantoms from the overworld as they are technically no longer needed because they are in the end and we can get membranes from end fishing. This could because they no longer have to swarm the overworld and we could remove the Phantom Scarecrow datapack which could help tps slightly.

**Comment by hannafoxie** [comment]
i approve the effort lmao

**Comment by aisukuma** [comment]
Based on a spark report, 50% of the ticks is from the `WorldServer` and 33% is from `CustomFunctionData`
- With the downtime happening next week, I'm hoping this should solve some of the ticks in `WorldServer` (but I won't be able to finish the overworld pregen from that downtime, will need approximately 1.5 days to finish)
- With the `CustomFunctionData`, I'm assuming its coming from datapacks, so I'll slowly disable some datapacks to see if that does anything
